lib/simple-xmpp.js

```diff
@@ -46,7 +47,6 @@ function SimpleXMPP() {
     //setting status here
     this.STATUS = STATUS;
     var self = this;
-    this.Element = xmpp.Element;
```

Comment:
moved to SimpleXMPP prototype

I recommend to move other commondmethods/properties on the prototype as well